Library/Homebrew/.rubocop.yml

```diff
@@ -37,7 +37,7 @@ Metrics/PerceivedComplexity:
 Metrics/MethodLength:
   Max: 260
 Metrics/ModuleLength:
-  Max: 600
+  Max: 650
```

Comment:
Please reduce the length rather than bumping this.

Reply:
Would it be okay to split `utils/github` into separate modules to fix this?

Reply:
@nandahkrishna Yeh, that makes sense to me 👍🏻.